openedx/core/djangoapps/content_libraries/tests/test_content_libraries.py

```diff
@@ -48,7 +48,9 @@ def test_library_crud(self):
         Test Create, Read, Update, and Delete of a Content Library
         """
         # Create:
-        lib = self._create_library(slug="lib-crud", title="A Test Library", description="Just Testing")
+        lib = self._create_library(
```

Comment:
Add a license check for the default license field too in `test_library_alternative_type` below, while we're at it?